### PR TITLE
Deploy Staging: optimism, arbitrum, binance, polygon

### PR DESCRIPTION
**Pull request by anna-carroll**

### CODE REVIEW

**Review by anna-carroll — COMMENTED** [review]
WIP review, but: 

- attestation keys, not signer keys
- staging `recoveryTimelock` should be 180
- naming for networks should match the provisioned keys in infrastructure

File: pending-deploy/staging.json [hunk 1]
```diff
@@ -34,60 +34,7 @@
...
-        },
-        "subsidizedRemotes": [
-          "evmostestnet",
-          "kovan",
-          "milkomedaC1testnet",
-          "mumbai",
-          "neontestnet",
-          "rinkeby"
-        ]
+        "subsidizedRemotes": ["goerli", "neontestnet", "rinkeby"]
```
[the remark this answers is not in the document]
Comment (anna-carroll) [reply pending-deploy/staging.json:37]: let's revisit agents and bridge config later

File: staging.json [hunk 2]
```diff
@@ -376,26 +606,152 @@
...
+        "bsc-testnet": {
+          "beacon": "0xd8be3c0d3ce1c077184a931cc88f938ec5e639de",
+          "implementation": "0xa179a65273b55a424ddb2834ce3983f4b5d6a5fb",
+          "proxy": "0x32641eb9af76a6d62e44d09e7a7f02f6ff5bc5d7"
+        },
         "evmostestnet": {
-          "beacon": "0xd8Be3C0d3CE1C077184a931cc88F938Ec5E639dE",
-          "implementation": "0xA179A65273b55A424DDB2834ce3983f4b5d6A5Fb",
-          "proxy": "0x41737AC8A129F20707D914ff088a7eC3e57bC99B"
+          "beacon": "0xd8be3c0d3ce1c077184a931cc88f938ec5e639de",
```
Comment (anna-carroll) [inline staging.json:620]: note to self: why did it overwrite some existing replicas?

how will that affect the system?

Comment (anna-carroll) [reply staging.json]: it was actually just different case of the same address, all good

**Review by luketchang — APPROVED** [review]
checks pass in rust repo

**Pull request merged**